commune_app/all_models/communes.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from .users import User
+from .chores import Chore
+from .votes import Vote
+
+
+class Commune(models.Model):
+    """
+      add something
+    """
+    id = models.IntegerField(unique=True, primary_key=True, auto_created=True)
```

Comment:
I think it should be CharField to allow IDs that start with 0

Reply:
@NicolePauzner 
what you mean? and what changes are you asking for?

Reply:
Change IntegerField to CharField so that an id that starts with the digit 0 will be legal -> 012345678 won't turn out to be 12345678 or raise an error

Reply:
@ofirtzabari if you want the default id which is auto_created 
you also can remove this line and you will see it in the migration file